lib/execution_engine2/db/models/transfer/transfer_ujs_njs.py

```diff
@@ -2,13 +2,21 @@
 import datetime
 import os
 from configparser import ConfigParser
+
 from pymongo import MongoClient
+from mongoengine import  connect
 
 jobs_database_name = "ee2_jobs"
 
+epoch = datetime.datetime.utcfromtimestamp(0)
+
+def unix_time_millis(dt):
+        if dt is None:
+            return 0
+        return (dt - epoch).total_seconds() * 1000.0
 
 try:
-    from lib.execution_engine2.db.models.models import (
+    from .execution_engine2.db.models.models import (
```

Comment:
I tried both of these imports and running the script, but it doesn't work. Do I need to try from a specific directory or something?